pkg/arvo/app/contact-store.hoon

```diff
@@ -126,6 +126,14 @@
               !=(contact(last-updated *@da) u.old(last-updated *@da))
           ==
         [~ state]
+      ~|  "cannot add a data url to cover!"
+      ?>  ?|  ?=(~ cover.contact)
```

Comment:
What's the point of this check?

Reply:
Either there is no cover image OR we check to make sure the cover image does not begin with the data url schema